hnginternship5/ziki
# #165424301 Added Published Post Page

**PURPOSE OF PR**
Implement UI of the published post page

**FEATURES IMPLEMENTED**
Light mode, Dark Mode and responsive design

**HOW TO TEST**
Append dark to body class to toggle dark mode and use the web console to check how responsive the UI is 

**PIVOTAL TRACKER STORY LINK**
https://www.pivotaltracker.com/n/projects/2327725/#165424301
